pkg/policy/selectorcache.go

```diff
@@ -244,9 +244,6 @@ func (s *selectorManager) Equal(b *selectorManager) bool {
 // of the selections. If the old version is returned, the user is
 // guaranteed to receive a notification including the update.
 func (s *selectorManager) GetSelections() []identity.NumericIdentity {
```

Comment:
This can be squashed with commit "policy: add interface for receiving updates on starting / stopping us"

Reply:
@aanm I am going to squash once I get approvals from all parties - I am trying to make reviews easier in that people only have to look at follow-up commits after their first round of comments.